x/ibc/core/02-client/types/genesis.go

```diff
@@ -103,11 +103,15 @@ func (gs GenesisState) UnpackInterfaces(unpacker codectypes.AnyUnpacker) error {
 // Validate performs basic genesis state validation returning an error upon any
 // failure.
 func (gs GenesisState) Validate() error {
+	// keep track of the max sequence to ensure it is less than
+	// the next sequence used in creating client identifers.
+	var maxSequence uint64 = 0
+
 	if err := gs.Params.Validate(); err != nil {
 		return err
 	}
 
-	validClients := make(map[string]bool)
+	validClients := make(map[string]exported.ClientState)
```

Comment:
If you're only using this for getting clienttype, you may as well make it `map[string]string` and store client type in value

Reply:
sure